docs/components/OpenAPI.md

```diff
@@ -0,0 +1,64 @@
+Overview
+========
+                              
+Nussknacker can use services documented with OpenAPI specification.
+We use Swagger to parse OpenAPI, versions 2.x and 3.x are supported 
+(version 2.x should be considered as deprecated).
+
+Nussknacker applies following rules when mapping OpenAPI services to enrichers:
+- Each operation is mapped to one enricher
+- If operationId is configured it's used as enricher name, otherwise we create it as concatenation
+of HTTP method, path and parameters (to provide some level of uniqueness)
+- Parameters (path, query of header) are used to define enricher parameters
+- If specification declares body parameter as object, we expand to parameter list  
+- We expect operation to define 200/201 response, returned object is the one that is the result of enricher
+- We map 404 HTTP code to null value of enricher
+- We use `externalDocs.url` to extract documentation link
+
+Table below describes data types that OpenAPI integration handles:
```

Comment:
I've found greate Intelij Plugin for formatting markdown tables https://plugins.jetbrains.com/plugin/7550-pipe-table-formatter/reviews

Reply:
:100: thanks :)